packages/react-native-bridge/ios/SourceFile.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+internal struct SourceFile {
```

Comment:
I separated this out to its own file because it now becomes useful for reading the `supported-blocks` Json file as well